packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_service_runs.py

```diff
@@ -117,9 +117,40 @@ class ResourceTrackerServiceRunStatus(str, enum.Enum):
     sa.Column(
         "node_name",
         sa.String,
-        nullable=True,
+        nullable=False,
```

Comment:
I like these changes, but should it be checked that this is doable for all the data in the db on the different deployments? Could it be that there is data around which would not allow this?

Reply:
Yes, I have checked it upfront, they are even defined as non-nullable in the `projects` table.